scheduler.go

```diff
@@ -619,9 +619,9 @@ func (s *Scheduler) removeByCondition(shouldRemove func(*Job) bool) {
 	s.setJobs(retainedJobs)
 }
 
-// RemoveByTag will remove a job by a given tag.
-func (s *Scheduler) RemoveByTag(tag string) error {
-	jobs, err := s.findJobsByTag(tag)
+// RemoveByTag will remove Jobs that match all given tags.
+func (s *Scheduler) RemoveByTag(tags ...string) error {
```

Comment:
Will you add `RemoveByTags` as a separate function and then have it simply call `RemoveByTag`. Just provides the same grammatical syntax that the library follows for `Second` and `Seconds` e.g.

Reply:
Good idea, I'll add that 👍